packages/components/tests/dummy/app/templates/components/accordion.hbs

```diff
@@ -0,0 +1,330 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Accordion Component"}}
```

Comment:
general consideration: maybe I would try to see if makes sense to showcase the "base elements" in isolation too, similar to what we do for example for the [Pagination](https://hds-showcase.vercel.app/components/pagination), [Tabs](https://hds-showcase.vercel.app/components/tabs),

Reply:
I added base elements to the Showcase and made some changes to the button component to get states, etc. working. Let me know if you'd like me to reorganize the Showcase to move any of the sections or anything.

Btw, the standalone examples of `Accordion::Item::Button` states don't work great since they require a container with position:relative to contain the pseudo element for the focus ring so let me know if I should make changes to those examples or perhaps just delete them.

Reply:
Made a number of follow on improvements to Showcase after pairing session w/ @didoo